SentinelOne/sentinelone_module/logs/configuration.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+from uuid import UUID
+
+from sekoia_automation.connector import DefaultConnectorConfiguration
+
+
+class SentinelOneLogsConnectorConfiguration(DefaultConnectorConfiguration):
+    client_secret: str
```

Comment:
For secrets, you should specify `Field(secret=True,...)` like [this](https://github.com/SEKOIA-IO/automation-library/blob/main/Checkpoint/connectors/__init__.py#L11)
```suggestion
    client_secret: str = Field(secret=True, description="The client secret for the authentication")
```
Suggestion to apply only if the client_secret and client_id are useful.
:warning: Don't forget to import Field from pydantic